conda/cli/main.py

```diff
@@ -113,7 +113,7 @@ def init_loggers(context=None):
     initialize_logging()
     if context and context.json:
         # Silence logging info to avoid interfering with JSON output
-        for logger in ('print', 'stdoutlog', 'stderrlog'):
+        for logger in ('conda.stdout.verbose', 'conda.stdoutlog', 'conda.stderrlog'):
```

Comment:
As long as we make sure `conda.stderrlog` is always going to stderr, then we don't have to silence it for the `--json` flag.  At least, the pattern we've followed so far is that, when the `--json` flag is enabled, everything written to stdout should be json, while stderr can still have non-json.

And some further information I guess... That old conda/console.py module made it so individual json payloads were always delimited with a `\0` character.  We should probably make sure we're enforcing that with our loggers now too.